src/test/java/DukeTest.java

```diff
@@ -0,0 +1,59 @@
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.io.FileWriter;
+import java.io.IOException;
+
+import org.junit.jupiter.api.Test;
+
+public class DukeTest {
+    // Overwrite "stored.txt" to start from clean slate
+    private FileWriter fw = new FileWriter("stored.txt");
+
+    private ListOfItems listOfItems = new ListOfItems();
+    private HandleFile handleFile = new HandleFile(listOfItems);
+    private Parser parser = new Parser(listOfItems, handleFile);
+
+    public DukeTest() throws IOException {
+    }
+
+    @Test
+    void testTodo() {
```

Comment:
Maybe using the three part format for naming test methods can provide more clarity on what you are testing.